tomcat/src/main/java/org/apache/coyote/http11/request/HttpHeaders.java

```diff
@@ -0,0 +1,45 @@
+package org.apache.coyote.http11.request;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class HttpHeaders {
+
+    private final Map<String, String> payLoads;
+
+    public HttpHeaders(Map<String, String> payLoads) {
+        this.payLoads = payLoads;
+    }
+
+    public static HttpHeaders readRequestHeader(BufferedReader bufferedReader) throws IOException {
+        Map<String, String> payLoads = new HashMap<>();
+        String line;
+        while ((line = bufferedReader.readLine()) != null) {
+            if (line.isBlank()) {
+                break;
+            }
+
+            String[] split = line.split(":");
+
+            if(split.length != 2) {
+                continue;
+            }
+
+            String key = split[0].trim();
+            String value = split[1].trim();
+            payLoads.put(key, value);
+        }
+
+        return new HttpHeaders(payLoads);
+    }
```

Comment:
[제안]
stream의 takeWhile문을 활용해 아래처럼 가독성이 좀 더 좋도록 작성할 수도 있을 꺼 같아요 😄
```suggestion
public static HttpHeaders readRequestHeader(BufferedReader bufferedReader) throws IOException {
        Map<String, String> headers = bufferedReader.lines()
                .takeWhile(line -> !line.isBlank())
                .map(line -> line.split(":", 2))
                .collect(Collectors.toMap(
                        split -> split[0].trim(),
                        split -> split[1].trim()
                ));

        return new HttpHeaders(headers);
     }
```

Reply:
stream에 takeWhile 문이 있는지 처음 알았어요!  맛있네요 🍗